src/promptflow-azure/promptflow/azure/_storage/cosmosdb/span.py

```diff
@@ -36,8 +39,8 @@ def __init__(self, span: SpanEntity, collection_id: str, created_by: dict) -> No
         self.context = span.context
         self.kind = span.kind
         self.parent_id = span.parent_id
-        self.start_time = span.start_time.isoformat()
-        self.end_time = span.end_time.isoformat()
+        self.start_time = span.start_time.strftime(DATE_FORMAT)
```

Comment:
Also make similar change for Summary.py?
https://github.com/microsoft/promptflow/blob/main/src/promptflow-azure/promptflow/azure/_storage/cosmosdb/summary.py#L176

Reply:
That would be an easy change, which should not be in this PR. Will commit in another PR.